apps/web/src/lib/workflows.ts

```diff
@@ -52,11 +59,26 @@ export const defaultWorkflowSummary: WorkflowSummary = {
 export function listItemToWorkflowSummary(
   item: WorkflowListItem,
 ): WorkflowSummary {
+  const triggerType = item.trigger_type === "scheduled" 
+    ? "Scheduled" 
+    : item.trigger_type === "webhook" 
+    ? "Webhook" 
+    : "Manual";
+  
+  // Determine status based on is_active flag (defaults to inactive if no schedule)
+  const status: "active" | "inactive" = item.schedule?.is_active ? "active" : "inactive";
```

Comment:
```suggestion
  const status = item.schedule?.is_active ? "active" : "inactive";
```
TypeScript will automatically infer the union type "active" | "inactive" here, so we can remove the explicit type annotation to keep it cleaner.